build-sensor/src/Main.cpp

```diff
@@ -0,0 +1,24 @@
+//
```

Comment:
Does this work out-of-the-box? We're re-defining a few things along the way, I think?

Reply:
Yes, it will build and compile and run OOTB.